soul-admin/pom.xml

```diff
@@ -169,6 +169,17 @@
             <version>1.4.200</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.undertow</groupId>
+            <artifactId>undertow-core</artifactId>
+            <version>${undertow.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.java-websocket</groupId>
+            <artifactId>Java-WebSocket</artifactId>
+            <version>1.5.0</version>
```

Comment:
I have a question about this.I think java-WebSockert  is imported by test case here, why not add test scope .And I think name the version as a property like undertow is better.

Reply:
sry, i forgot to test scope. The problem of version i will fix it with other pom file.